db/bigtable_eth1.go

```diff
@@ -2252,9 +2260,11 @@ func (bigtable *Bigtable) GetArbitraryTokenTransfersForTransaction(transaction [
 	tokensToAdd := make(map[string]*types.ERC20Metadata)
 	// init
 	for _, t := range transfers {
-		names[string(t.From)] = ""
-		names[string(t.To)] = ""
-		tokens[string(t.TokenAddress)] = nil
+		if t != nil {
```

Comment:
This should not suffice as the same `transfers` are accessed here anyways again:
https://github.com/gobitfly/eth2-beaconchain-explorer/blob/2f7566c921508e73a435184ee5d86c78258e95f3/db/bigtable_eth1.go#L2315